scrapy/pipelines/images.py

```diff
@@ -129,15 +132,33 @@ def get_images(self, response, request, info, *, item=None):
                                  f"({width}x{height} < "
                                  f"{self.min_width}x{self.min_height})")
 
-        image, buf = self.convert_image(orig_image)
+        if self._deprecated_convert_image is None:
+            self._deprecated_convert_image = 'response_body' not in get_func_args(self.convert_image)
+            if self._deprecated_convert_image:
+                warnings.warn('ImagesPipeline.convert_image() method overriden in a deprecated way, '
```

Comment:
What do you think about using the exact class name here instead of ImagesPipeline?

Reply:
The same applies for the other warninig message

Reply:
By exact class name you mean the name of the class instance variable? I didn't quite get that.

Reply:
The warning would fire if user creates a subclass of ImagesPipeline and overrides convert_image method. So it would be `MyImagesPipeline.convert_image` method which we should warn about, not `ImagesPipeline.convert_image`. Something like `self.__class__.__name__` + ".convert_images()" (haven't checked it).